packages/flutter_tools/lib/src/ios/migrations/remove_bitcode_migration.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../xcode_project.dart';
+
+/// Remove deprecated bitcode build setting.
+class RemoveBitcodeMigration extends ProjectMigrator {
+  RemoveBitcodeMigration(
+    IosProject project,
+    super.logger,
+  )   : _xcodeProjectInfoFile = project.xcodeProjectInfoFile;
+
+  final File _xcodeProjectInfoFile;
+
+  @override
+  bool migrate() {
+    if (_xcodeProjectInfoFile.existsSync()) {
+      processFileLines(_xcodeProjectInfoFile);
+    } else {
+      logger.printTrace('Xcode project not found, skipping removing bitcode migration.');
```

Comment:
isn't `migrate()` supposed to return false if the migration was skipped?

Reply:
At least how it works now is that returning `false` means something catastrophic happened and the build fails:
https://github.com/flutter/flutter/blob/5f5d4802fe7743ad0d26d2b8fe1fdd0de9dad8ca/packages/flutter_tools/lib/src/linux/build_linux.dart#L50-L52
https://github.com/flutter/flutter/blob/70aaff121731846b6218273424680c215da2d02f/packages/flutter_tools/lib/src/ios/mac.dart#L134-L136
etc
https://github.com/flutter/flutter/blob/c77e4cdd34f1b97703869d02b238d3c99bae9873/packages/flutter_tools/lib/src/base/project_migrator.dart#L85-L90

So this comment is just wrong:
https://github.com/flutter/flutter/blob/c77e4cdd34f1b97703869d02b238d3c99bae9873/packages/flutter_tools/lib/src/base/project_migrator.dart#L19-L20

Reply:
Will fix in follow-up PR.